src/preview.js

```diff
@@ -1,51 +1,70 @@
-import countries from '../assets/countries.json';
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+ import countries from '../assets/countries.json';
 import continentNames from '../assets/continent-names.json';
 import continents from '../assets/continents.json';
 import { getEmojiFlag } from './utils';
-import { __, sprintf } from '@wordpress/i18n';
 
 export default function Preview( { countryCode, relatedPosts } ) {
 	if ( ! countryCode ) return null;
 
 	const emojiFlag = getEmojiFlag( countryCode ),
-	      hasRelatedPosts = relatedPosts?.length > 0;
+		hasRelatedPosts = relatedPosts?.length > 0;
 
 	return (
 		<div className="xwp-country-card">
-			<div className="xwp-country-card__media" data-emoji-flag={ emojiFlag }>
-				<div className="xwp-country-card-flag">
-					{ emojiFlag }
-				</div>
+			<div
+				className="xwp-country-card__media"
+				data-emoji-flag={ emojiFlag }
+			>
+				<div className="xwp-country-card-flag">{ emojiFlag }</div>
 			</div>
 			<h3 className="xwp-country-card__heading">
-				{ __( 'Hello from' ) }
-				{ ' ' }
-				<strong>{ countries[countryCode] }</strong>
-				{ ' ' }
-				(<span className="xwp-country-card__country-code">{ countryCode }</span>),
-				{ ' ' }
-				{ continentNames[continents[countryCode]] }!
+				{ __( 'Hello from', 'xwp-country-card' ) }{ ' ' }
+				<strong>{ countries[ countryCode ] }</strong> (
+				<span className="xwp-country-card__country-code">
+					{ countryCode }
+				</span>
+				), { continentNames[ continents[ countryCode ] ] }!
 			</h3>
 			<div className="xwp-country-card__related-posts">
-				<h3 className="xwp-country-card__related-posts__heading">
-					{ hasRelatedPosts ? sprintf( __( 'There are %d related posts:' ), relatedPosts.length ) : __( 'There are no related posts.' ) }
-				</h3>
+				<h4 className="xwp-country-card__related-posts__heading">
+					{ hasRelatedPosts
+						? sprintf(
+								/** Translators: %d is the number of posts found. */
+								__(
+									'There are %d related posts:',
+									'xwp-country-card'
+								),
+								relatedPosts.length
+						  )
+						: __(
+								'There are no related posts.',
+								'xwp-country-card'
+						  ) }
+				</h4>
 				{ hasRelatedPosts && (
 					<ul className="xwp-country-card__related-posts-list">
 						{ relatedPosts.map( ( relatedPost, index ) => (
 							<li key={ index } className="related-post">
```

Comment:
Since we have a unique identifier for a related post handy, we should not be using a map's `index`:

```suggestion
							<li key={ relatedPost.id } className="related-post">
```